html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+<body>
+  <h1>Coding Form!</h1>
+  <form id="usrform">
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" id="email">
+    </div>
+    <div>
+      <label for="code">Why is coding cool?</label>
+    </div>
+    <div>
+      <select id="code">
+        <option value="select an option">Select an option</option>
+        <option value="because it's interesting">Because it's interesting</option>
+        <option value="because it's fun">Because it's fun</option>
+        <option value="because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" id="html">
+    <label for="html">HTML</label>
+    <input type="radio" id="css">
+    <label for="css">CSS</label>
+    <input type="radio" id="javascript">
```

Comment:
Let's give these `input`s a `value` and a `name` attribute as well, again this is for properly passing the data during form submission. 

Hint: the value of `name` should be the same for all these inputs since they are all options to the same question.